node-require/calculate.js

```diff
@@ -0,0 +1,21 @@
+const add = require('./add');
+const subtract = require('./subtract');
+const multiply = require('./multiply');
+const divide = require('./divide');
+
+const num1 = process.argv[2];
+const operator = process.argv[3];
+const num2 = process.argv[4];
```

Comment:
To avoid having to `parseInt` in every function it would be better to parse the input here so you only have to do it here and not in every function.

Reply:
much better. less > more.